.lintrunner.toml

```diff
@@ -151,7 +151,8 @@ command = [
     'lintrunner_adapters',
     'run',
     'grep_linter',
-    '--pattern= Executorch\W+',
```

Comment:
I think the space here may be important. Though `\bExecutorch` may be more of what was intended.